napari/layers/points/_tests/test_points_mouse_bindings.py

```diff
@@ -470,6 +470,20 @@ def test_unselecting_points(create_known_points_layer_2d):
     # Check clicked point selected
     assert len(layer.selected_data) == 0
 
+    # check that this also works with scaled data and position near a point (see #5737)
```

Comment:
Good idea, I'd like to see this check implemented before merging.

Reply:
It is, just below this comment :P

Reply:
Oh, duh! Sorry 🤣